hvac/v1/__init__.py

```diff
@@ -43,6 +43,8 @@ def __init__(self, url='http://localhost:8200', token=None,
         :param adapter: Optional class to be used for performing requests. If none is provided, defaults to
             hvac.adapters.Request
         :type adapter: hvac.adapters.Adapter
+        :param namespace: Optional Vault Namespace.
```

Comment:
Apologies again for missing a bit, but looks like we still need to propagate this kwarg into the `adapters.Request` instantiation below for the parameter to have any effect on the subsequent requests.